MapboxCoreNavigationTests/TunnelIntersectionManagerTests.swift

```diff
@@ -140,9 +140,8 @@ class TunnelIntersectionManagerTests: XCTestCase {
         routeController.locationManager(routeController.locationManager, didUpdateLocations: [tunnelLocation])
         
         // Enable the tunnel animation, which should enable the simulated location manager
-        routeController.tunnelIntersectionManager?.delegate?.tunnelIntersectionManager?(routeController.locationManager, willEnableAnimationAt: routeController.location!) { isAnimationEnabled, manager in
-                XCTAssertTrue(isAnimationEnabled, "Animation through tunnel should be enabled.")
```

Comment:
@1ec5 the reason why this block was never called is animationEnabled was set to true in the previous location update. In fact, upon the recent decisions to keep both location managers active this means now we don't have a delay in location updates as we did previously.